happytransformer/runners/runner_answer_question.py

```diff
@@ -0,0 +1,172 @@
+import torch
+
+from collections import namedtuple
+
+
+class AnswerQuestionRunner():
+    SumPair = namedtuple('SumPair', ['idx1', 'idx2', 'sum'])
```

Comment:
Not sure why this type is stored within the class

Reply:
Moved to runner_util.py